File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int,int> v;
        for(int i=0;i<nums.size();i++)
        {
            if(v.find(target-nums[i])!=v.end()) return {i,v[target-nums[i]]};
            v[nums[i]]=i;
        }
        return {};
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        map={}
        for i in range(len(nums)):
            c=target-nums[i]
            if c in map:
                return [map[c],i]
            map[nums[i]]=i
        return[]


        